brion/plugin/spikeReportBinary.cpp

```diff
@@ -115,6 +115,9 @@ SpikeReportBinary::SpikeReportBinary( const SpikeReportInitData& initData )
         _memFile.reset( new BinaryReportMap( getURI().getPath( )));
     else
         _memFile.reset( new BinaryReportMap( getURI().getPath(), 0 ));
+
+    if( _memFile->getNSpikes() > 0 )
+        _currentTime = _memFile->getReadableSpikes()[0].first;
```

Comment:
This is the only place I find really acceptable to replace -inf by something else, but you have to update the specification of getCurrentTime accordingly